aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -144,4 +165,12 @@ and will generate HTML as follows.
 
 (of course the domain, in the case above is localhost, but this will be substituted for whatever the actual domain hosting the web site is hosted at).
```

Comment:
> (of course the domain, in the case above is localhost, but this will be substituted for whatever the actual domain hosting the web site is hosted at).

The domain in the example is `localhost`, but the Anchor Tag Helper uses the website's public domain when generating the URL.